src/main/cpp/benchmarks/CMakeLists.txt

```diff
@@ -14,9 +14,22 @@
 # limitations under the License.
 #=============================================================================
 
-# this library isn't exposed by cudf, so we have to grab it this way
-find_library(CUDF_DATAGEN "libcudf_datagen.a" REQUIRED NO_DEFAULT_PATH
-    HINTS "${CUDF_DIR}/cpp/build/benchmarks"
+add_library(DATAGEN_LIB STATIC common/generate_input.cu)
```

Comment:
I'm assuming this generates a library named libDATAGEN_LIB.a, and we don't normally use capital letters in library names.  Normally caps are used for CMake variable names, but that's not the case here.  Following the convention used in cudf, I suggest using a name like `spark_rapids_jni_datagen` here.